sympy/solvers/ode.py

```diff
@@ -3965,6 +4028,43 @@ def ode_2nd_power_series_regular(eq, func, order, match):
             return Eq(f(x), collect(finalseries1 + finalseries2,
                 [C0, C1]) + Order(x**terms))
 
+def ode_2nd_linear_bessel(eq, func, order, match):
```

Comment:
These will need to be documented in doc/src/modules/solvers/ode.rst

Reply:
okay